### PR TITLE
update readme to mention 4.3

### PR DESCRIPTION
**Pull request by ctzsnooze**
Points a 4.3 user to the nightly builds, needed until we have a 3.6.0 release for 4.3.

Also some minor changes to the notes about building with yarn.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
